src/intents/DSL/actions/limitSwapAction.ts

```diff
@@ -13,7 +13,7 @@ function limitSwapAction ({
   tokenInAmount,
   tokenOutAmount
 }: LimitSwapActionArgs): SegmentArgs[] {
-  if (tokenOutAmount === undefined) {
+  if (tokenOutAmount === undefined || tokenOutAmount === null) {
```

Comment:
I think an `if (!tokenOutAmount)` check here would be better, since it covers undefined, null, and more cases where tokenOutAmount is not set

Reply:
Done!